Resources/Prototypes/_Impstation/Entities/Objects/Specific/Robotics/Borg/borg_modules.yml

```
@@ -133,3 +133,20 @@
     - RadioHandheld
   - type: BorgModuleIcon
     icon: { sprite: _Impstation/Interface/Actions/actions_borg.rsi, state: basic-mining-module }
+
+- type: entity
+  id: BorgModuleCourier
+  parent: [ BaseBorgModuleCargo, BaseProviderBorgModule ]
+  name: courier cyborg module
+  components:
+  - type: Sprite
+    sprite: _Impstation/Objects/Specific/Robotics/borgmodule.rsi
```

missing as s after borgmodule.
```suggestion
    sprite: _Impstation/Objects/Specific/Robotics/borgmodules.rsi
```